### PR TITLE
Upgrade to 2.17

### PR DESCRIPTION
**Pull request by snewcomer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,20 +38,17 @@ matrix:
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH=$HOME/.yarn/bin:$PATH
-  - if [[ $(npm -v | cut -d '.' -f 1) -lt 3 ]]; then npm i -g npm@^3; fi
-  - yarn global add bower
-  - bower install
```
Comment (joshsmith) [inline .travis.yml]: I think the `before_install` lines are needed for us.

File: .travis.yml [hunk 2]
```diff
@@ -8,23 +8,23 @@ node_js:
 sudo: false
 dist: trusty
 
-cache:
-  yarn: true
-  directories:
-    - $HOME/.cache # includes bowers cache
```
Comment (joshsmith) [inline .travis.yml]: Do we use anything in bower?

File: .travis.yml [hunk 3]
```diff
@@ -38,20 +38,17 @@ matrix:
...
-  - bower install
 
 install:
   - yarn install --no-lockfile --non-interactive
 
 script:
   # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
-  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO --skip-cleanup
```
Comment (joshsmith) [inline .travis.yml]: Does this need the `test` still or no?

File: .travis.yml (hunk 2 above)
Comment (snewcomer) [reply .travis.yml]: nope

File: .travis.yml (hunk 3 above)
Comment (snewcomer) [reply .travis.yml]: These were the defaults from ember-cli

**Comment by joshsmith** [comment]
🙌

**Pull request merged**